__tests__/Component-test.tsx

```diff
@@ -1,18 +1,27 @@
-import React from "react";
-import {fireEvent, render} from "@testing-library/react-native"
-import ShortGoalsComponent from "../src/components/ShortGoalsComponent/ShortGoalsComponent";
+import React from 'react';
+import { fireEvent, render } from '@testing-library/react-native';
+import ShortGoalsComponent from '../src/components/ShortGoalsComponent/ShortGoalsComponent';
```

Comment:
+1

Reply:
it's done by prettier, marking this and the previous ones resolved